atest/robot/cli/dryrun/if.robot

```diff
@@ -7,17 +7,17 @@ Resource          dryrun_resource.robot
 IF will not recurse in dry run
     ${tc}=    Check Test Case    ${TESTNAME}
     Check Branch Statuses    ${tc.body[0]}                            Recursive if         PASS
-    Check Branch Statuses    ${tc.body[0].body[0].body[0].body[0]}    Recursive if         NOT RUN
+    Check Branch Statuses    ${tc[0,0,0,0]}    Recursive if         NOT RUN
```

Comment:
I believe that `0, 0, 0, 0` would be better than `0,0,0,0` but don't feel too strongly about that. We can discuss on Slack.